### PR TITLE
Contact form mobile version

### PR DESCRIPTION
**Pull request by nstrvs**
Create the contact form and link it to Formspree.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -159,5 +159,26 @@ <h3>Skills</h3>
...
+        </label>
+        
+        <label>
+          <span>Your email</span>
+          <input type="email" name="_replyto" required>
+        </label>
+        <textarea name="message" maxlength="500" placeholder="Enter your message here..."></textarea>
+        <button type="submit">Start collaboration</button>
+      </form>
+    </section>
```
Comment (Amrhub) [inline index.html:182]: - [ ] You implemented functionality perfectly fine, but we need to work on our styling to match The Figma design 

Your Implementation | [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=67%3A1095)
:----:|:-----:
image | image

Please let me know if you have any troubles made you not implement that at all.

**Review by Amrhub — CHANGES_REQUESTED** [review]
Hi @nstrvs @41hulk,  👋🏻 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!
These changes aim to add some awesomeness to your awesome project and code skills.

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by nstrvs** [comment]
Thanks for your suggestion @Amrhub, but I used media queries of 320px width (accordingly to the figma design), that's why you didn't see any design. 

Please check that before giving a review.

**Comment by mke2111** [comment]
Project Under Second Opinion Review.
Kindly wait for a response to come your way. 🙏🏽

**Review by mke2111 — COMMENTED** [review]
Hello  @nstrvs & @41hulk 👋,

_This is an additional review as a result of the second opinion request._

Your project is taking shape and looks great based on what you have done so far. 👍🏽 Keep going 😉 

For mobile-first approach, think of these mobile screen sizes ranging from 300px - 516px/768px. So you're limiting your styles to apply to only the screen sizes that are 320px and below and it's the reason the styles break(don't apply) once the width is above 320px. What you want is a max-width of 768px and below.

Make the requested change to cater to all mobile screen sizes using a max-width of 768px. Submit another code review request (CRR) once you are ready and somebody will give a review.

Keep rocking 🚀 and Happy Coding 💻

**Review by Deepakdanger — APPROVED** [review]
Hi @nstrvs, 

### Status: Approved ✔️

Well Done implementing on the suggested changes. Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 

Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**